src/serlio/modifiers/RuleAttributes.cpp

```diff
@@ -56,6 +56,13 @@ std::wstring getNiceName(const std::wstring& fqAttrName) {
 	return cleanForMaya(prtu::removeImport(prtu::removeStyle(fqAttrName)));
 }
 
+std::wstring getDuplicateCountSuffix(const std::wstring& mayaName, std::map<std::wstring, int>& mayaNameDuplicateCountMap) {
```

Comment:
Please move this function into the utils and write unit tests for it.

Reply:
Added unit test based on example in internal ticket